### DISCUSSION TITLE
About 1st session

### DISCUSSION THREAD

**Discussion by CyberWitch7** [community/events#22] [AI]
What will exactly be in the first session? Its my first time attending something like this😅

**Comment by samus-aran** [comment]
Hi @CyberWitch7 👋 

Each track (AI, Security, Community and Cloud) has a few sessions! The first track for **AI** specifically is **How GitHub Copilot can impact developer happiness** Speaker **Seth Juarez** (Program Manager, AI | Microsoft). Here's the description below.

GitHub Copilot represents a huge leap forward in developer productivity, but did you know it also contributes to developer happiness? Join Seth Juarez and some special guests to learn what Copilot is, how it works, how it makes developers' lives easier, happier, and more productive.

To create your customised schedule, check out the [FAQ](https://github.com/community/events/discussions/1) and its instructions!

**Reply by CyberWitch7** [reply]
Thanks! @samus-aran
